### PR TITLE
Mlops04

### PR DESCRIPTION
**Pull request by krokrob**
this adapts DE04 predict api to the MLOps module

### CODE REVIEW

File: model/tests/test_cloud_data.py [hunk 1]
```diff
@@ -0,0 +1,105 @@
+
```
Comment (krokrob) [inline model/tests/test_cloud_data.py:1]: this one I don't know why it is there

Comment (gmanchon) [reply model/tests/test_cloud_data.py:1]: I think it is an old file from way back when the tests where not in `tests/challenge/test_challenge.py` subdirs

Comment (gmanchon) [reply model/tests/test_cloud_data.py:1]: we can get rid of it and its little .txt friends

Comment (gmanchon) [reply model/tests/test_cloud_data.py:1]: `test_cloud_data*`

File: model/tests/test_cloud_data_bq_chunks.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
+BigQuery error in query operation: Error processing job 'kevin-
+bootcamp:bqjob_r42e2e4f1e8ec09a7_000001806f7bbacf_1': Not found: Dataset kevin-
+bootcamp:sample_taxifare was not found in location US
```
Comment (gmanchon) [inline model/tests/test_cloud_data_bq_chunks.txt:3]: bad test value ?

File: model/tests/test_cloud_data_create_table_source.txt [hunk 3]
```diff
@@ -0,0 +1,2 @@
+BigQuery error in ls operation: Not found: Dataset kevin-
+bootcamp:sample_taxifare
```
Comment (gmanchon) [inline model/tests/test_cloud_data_create_table_source.txt:2]: you can populate the dataset with https://github.com/lewagon/taxi-fare/blob/mlops/model/Makefile_teachers

File: model/tests/test_cloud_data.py (hunk 1 above)
Comment (krokrob) [reply model/tests/test_cloud_data.py]: deleted

File: model/tests/test_cloud_data_bq_chunks.txt (hunk 2 above)
Comment (krokrob) [reply model/tests/test_cloud_data_bq_chunks.txt]: deleted

File: model/tests/test_cloud_data_create_table_source.txt (hunk 3 above)
Comment (krokrob) [reply model/tests/test_cloud_data_create_table_source.txt]: deleted

**Pull request merged**